cmake/oss.cmake.in

```diff
@@ -324,3 +324,18 @@ ExternalProject_Add(protobuf
     -Dprotobuf_BUILD_TESTS=OFF
     ${common_cmake_args}
 )
+
+ExternalProject_Add(WasmEdge
```

Comment:
Could you add an entry to: https://github.com/redpanda-data/redpanda/blob/dev/licenses/third_party.md

Reply:
👍 done - also removed v8